src/utils/getChildItems.ts

```diff
@@ -0,0 +1,29 @@
+import { DefaultListCssClasses } from "../ListRenderer";
+import { ItemElement } from "../types/Elements";
+
+/**
+ * Get all child items of the passed list item
+ */
+export function getChildItems(element: HTMLElement): ItemElement[] | null {
```

Comment:
its not clear that this method return only first level Childs. Maybe we can add `isFirstLevel` param to it? Thus we can use it both for getting first-level and all-levels subitems.